src/test/ui/numbers-arithmetic/saturating-float-casts.rs

```diff
@@ -1,15 +1,19 @@
 // run-pass
 // Tests saturating float->int casts. See u128-as-f32.rs for the opposite direction.
-// compile-flags: -Z saturating-float-casts
```

Comment:
Currently this test tests the wrong thing -- it tests Miri, not codegen.
Add `-Zmir-opt-level=0` to turn that off.